docs/excel/custom-functions-save-state.md

```diff
@@ -0,0 +1,51 @@
+---
+ms.date: 04/25/2019
+description: Use `Office.Storage` to save state with custom functions. 
+title: Save and share state in custom functions (preview)
+localization_priority: Priority
+---
+
+## Save and share state in custom functions
```

Comment:
Should mention how long state is persisted. does it time out after a certain number of days? Can you clear it (like by clearing cache?) Can you rely on it (like can user reset it somehow and surprise your code). This could be a future doc item, but at some point we should add this info.

Reply:
That is a really good question, will have to ask Keyur. I will make a note to ask him